Tests/RichTextKitTests/Extensions/String+ParagraphTests.swift

```diff
@@ -10,20 +10,19 @@ import RichTextKit
 import XCTest
 
 final class String_ParagraphTests: XCTestCase {
-    
-    let none = "foo bar baz"
-    let single = "foo\nbar baz"
-    let multi = "foo\nbar\rbaz"
 
-    func currentResult(for string: String, from location: UInt) -> UInt {
+    private let none = "foo bar baz"
+    private let single = "foo\nbar baz"
+    private let multi = "foo\nbar\rbaz"
+
+    private func currentResult(for string: String, from location: UInt) -> UInt {
         string.findIndexOfCurrentParagraph(from: location)
     }
 
-    func nextTesult(for string: String, from location: UInt) -> UInt {
+    private func nextResult(for string: String, from location: UInt) -> UInt {
```

Comment:
Omg, all these spelling errors that I haven't noticed before 🫣

Reply:
Turning on spell check in Xcode might be your new friend 😄 - https://sarunw.com/posts/spell-checking-in-xcode/